.template/variants/web/package.json.rb

```diff
@@ -40,6 +43,13 @@
 run 'npm pkg set scripts.stylelint:fix="stylelint **/*.scss --color --fix"'
 run 'npm pkg set scripts.codebase="yarn eslint && yarn stylelint"'
 run 'npm pkg set scripts.codebase:fix="yarn eslint:fix && yarn stylelint:fix"'
+run 'npm pkg set scripts.lint:docs:yml="eslint docs/openapi --ext .yml --color"'
+run 'npm pkg set scripts.lint:docs:openapi="spectral lint docs/openapi/openapi.yml -F error"'
+run 'npm pkg set scripts.lint:docs:dev="npm lint:docs:yml && npm lint:docs:openapi"'
+run 'npm pkg set scripts.lint:docs:public="npm build:docs && eslint public/openapi.yml --color --no-ignore && spectral\
+ lint public/openapi.yml -F error"'
+run 'npm pkg set scripts.build:docs="swagger-cli bundle docs/openapi/openapi.yml --outfile public/openapi.yml --type\
+ yaml"'
```

Comment:
Same here, also having these command added from the `addons/openapi` will allow to dry this code.